content/en/docs/started/installing-kubeflow.md

```diff
@@ -122,6 +122,13 @@ Packaged distributions are developed and supported by their respective maintaine
         <td><a href="/docs/distributions/charmed/">Docs</a></td>
         <td><a href="https://charmed-kubeflow.io/docs/quickstart">External Website</a></td>
       </tr>
+        <td>Kubeflow for Oracle Container Engine for Kubernetes</td>
+        <td>Oracle</td>
+        <td>Oracle Cloud Infrastructure (OCI)</td>
+        <td>1.6</td>
+        <td>N/A</td>
+        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
+      
```

Comment:
```suggestion
      </tr>
      <tr>
        <td>Kubeflow for Oracle Container Engine for Kubernetes</td>
        <td>Oracle</td>
        <td>Oracle Cloud Infrastructure (OCI)</td>
        <td>1.6</td>
        <td>N/A</td>
        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
      </tr>
```

Reply:
preview https://deploy-preview-3353--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/ shows syntax issues, you need to wrap <td>'s in a <tr>